iota/bin/repl.py

```diff
@@ -43,16 +37,14 @@ def execute(self, api, **arguments):
         # If ``debug_requests`` is specified, log HTTP requests/responses.
         if debug_requests:
             # Inject a logger into the IOTA HTTP adapter.
+            # This will turn on logging for underlying httpx client as well
```

Comment:
```suggestion
            # This will turn on logging for underlying http client as well
```

Wasn't sure if that was a typo.